public/usage-examples/physics/bitmap_collision/bitmap_collision-simple-oop.cs

```diff
@@ -0,0 +1,47 @@
+using SplashKitSDK;
```

Comment:
Visual bug here on this codespace, I verified the code in the branch it includes the line. Can ignore this
`using static SplashKitSDK.SplashKit;`